src/sync-merkleroots.test.ts

```diff
@@ -0,0 +1,315 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  ErrorStaleLastEvaluatedKey,
+  ErrorSyncMerkleRootsTimeout,
+  ExclusiveStartKeyPage,
+} from '../dist/typescript-npm-package.cjs';
```

Comment:
```suggestion
import {
  ErrorStaleLastEvaluatedKey,
  ErrorSyncMerkleRootsTimeout,
} from './errors';
import {
  ExclusiveStartKeyPage
} from './types';
```

I believe, you shouldn't import compiled code into the tests 🤔 - or I'm missing something.

Reply:
Oh, you're right it was autoimported from my IDE i missed the fact it was from the built package. Thanks for noticing